config/crd/kustomization.yaml

```diff
@@ -16,13 +17,15 @@ patchesStrategicMerge:
 #- patches/webhook_in_tanzukubernetesreleases.yaml
 #- patches/webhook_in_capabilities.yaml
 #- patches/webhook_in_calicoconfigs.yaml
+#- patches/webhook_in_kappcontrollerconfigs.yaml
```

Comment:
We don't need to submit ```patches/webhook_in_kappcontrollerconfigs.yaml``` and ```patches/cainjection_in_kappcontrollerconfigs.yaml``` files for now

Reply:
It's commented out

Reply:
why add commented ones though?